config/files/usr/share/ublue-os/firstboot/yafti.yml

```diff
@@ -18,6 +18,7 @@ screens:
         This step will enable hardening for installed flatpaks.
       actions:
         - run: flatpak override --user --filesystem=host-os:ro --env=LD_PRELOAD=/var/run/host/usr/lib64/libhardened_malloc.so
+        - run: uarches=$(/var/run/host/usr/lib64/ld-linux-x86-64.so.2 --help | grep '(supported, searched)' | cut -d'v' -f2); bestuarch=${uarches:0:1}; if [ ! -z "${bestuarch}" ]; then flatpak override --user --env=LD_PRELOAD=/var/run/host/usr/lib64/glibc-hwcaps/x86-64-v$bestuarch/libhardened_malloc.so; fi
```

Comment:
shouldn't this replace the one above? also, why not just use your ujust command from the other PR?

Reply:
The reason I don't replace the original is my command's logic does nothing if  v2-4 support is not present. If it is present, causing both `flatpak override` commands to run, the second override harmlessly updates the LD_PRELOAD value.

Reply:
The reason to not simply use the ujust command is with yafti, the command is (essentially) automatically run, whereas ujust requires the user to specifically bother to run it. Since this script works correctly on any x86_64 device (as far as my synthetic tests report), I see no reason to not just run this automatically. But perhaps you merge the ujust first and let people test it in the wild before merging it to yafti, where more users (and perhaps less experienced users) will have it run.

Reply:
@spaceoden you misunderstood me. I'm not saying this doesn't belong in yafti, it does. I'm saying instead of copy and pasting the entire script, just put this:

`- run: ujust harden-flatpak`